sidecar/proxy/envoy/manager.go

```diff
@@ -565,60 +548,6 @@ func buildFS(ruleList []api.Rule, workingDir string) error {
 		return err
 	}
 
-	dirName, err := ioutil.TempDir(workingDir+runtimeVersionsPath, "")
-	if err != nil {
```

Comment:
It is documented. https://lyft.github.io/envoy/docs/configuration/http_filters/fault_filter.html#runtime. We can get rid of all runtimes if you want to, but then we will have to make sure to leverage Envoy's RDS. And make do with envoy hot reload whenever we tweak fault injection stuff.